lua/fastaction/init.lua

```diff
@@ -94,22 +95,33 @@ end
 ---               `nil` if the user aborted the dialog.
 function M.select(items, opts, on_choice)
     opts.format_item = opts.format_item or tostring
-    local used_keys = {}
+    local used_keys = vim.tbl_extend('force', {}, m.config.dismiss_keys)
     ---@type {name: string, key: string, item: any, order: integer}[]}
     local options = {}
 
     ---@type string[]
     local content = {}
 
+    local chars = 1
```

Comment:
Move this hunk to a function in keys.lua that accepts #items as a parameter and returns valid_keys list.